### ISSUE TITLE
coq: proof-shell-invisible-cmd-get-result doesn't work with Test

### ISSUE THREAD

**Issue by SkySkimmer** [ProofGeneral/PG#172]
If I do eg `Goal forall n : nat, n = n.` then run `(proof-shell-invisible-cmd-get-result "Show Intros.")` I get the result
```
"n
"
```
as expected but if I run `(proof-shell-invisible-cmd-get-result "Test Universe Polymorphism.")` I get 
```
"
"
```
and the expected "The universe polymorphism mode is off" appears in the *response* buffer.
After hitting `C-x C-e` a few times:
![Screenshot](https://i.imgur.com/Pi43ABH.png)

**Comment by Matafou** [comment]
Indeed, the answer of Show Intros is not surrounded by <infomsg> </infomsg> as for Test xxx.
There are two problems here:
1)Coq is not homogeneous.
2) PG should look into those <infomsg> if there is nothing else to take.

Am I right?
P.

**Comment by cpitclaudel** [comment]
Yup, that's a known problem.  Look at what company-coq does to work around this: https://github.com/cpitclaudel/company-coq/blob/master/company-coq.el#L857  Does this help?

**Comment by SkySkimmer** [comment]
That works, thanks. Feel free to close if you WONTFIX.

**Comment by cpitclaudel** [comment]
I won't have time to work on a fix, but I'd be happy to merge one.
